ed_sensor_integration/tools/fitter_viz.cpp

```diff
@@ -183,6 +183,53 @@ double getYaw(const geo::Mat3& rotation)
     return angles.yaw;
 }
 
+void drawLine(
+        cv::Mat& canvas, geo::Vec2 point1, geo::Vec2 point2, geo::Transform2 pose, float resolution, int origin_x, int origin_y, cv::Scalar color)
+{
+    // computing outer corner positions (in meters)
+    geo::Vec2 tf_point1 = pose * point1;
+    geo::Vec2 tf_point2 = pose * point2;
+
+    // position to pixels
+    int x_p1 = origin_x + (int)(tf_point1.x * resolution);
+    int y_p1 = origin_y - (int)(tf_point1.y * resolution);
+    int x_p2 = origin_x + (int)(tf_point2.x * resolution);
+    int y_p2 = origin_y - (int)(tf_point2.y * resolution);
+
+    if (x_p1 < 0 || x_p2 < 0|| x_p1 >= canvas.cols || x_p2 >= canvas.cols){
+        std::cout << "Entity: x-coordinate out of range" << std::endl;
+        return;
+    }
+    if (y_p1 < 0 || y_p2 < 0|| y_p1 >= canvas.rows || y_p2 >= canvas.rows){
+        std::cout << "Entity: y-coordinate out of range" << std::endl;
+        return;
+    }
+
+    // paint to screen
+    cv::Point point1_p(x_p1, y_p1);
+    cv::Point point2_p(x_p2, y_p2);
+    cv::line(canvas, point1_p, point2_p, color, 1);
+}
+
+void drawShape2D(cv::Mat& canvas, const Shape2D& shape, geo::Transform2 pose, float resolution, int origin_x, int origin_y, cv::Scalar color)
+{
+    for (int i=0; i < shape.size(); i++){
+
+        for (int j=0; j < shape[i].size()-1; j++){
+            drawLine(canvas, shape[i][j], shape[i][j+1], pose, resolution, origin_x, origin_y, color);
+        }
+        drawLine(canvas, shape[i][0], shape[i][shape[0].size()-1], pose, resolution, origin_x, origin_y, color);
+    }
```

Comment:
```suggestion
    for (int i=0; i < shape.size(); i++)
    {
        for (int j=0; j < shape[i].size()-1; j++)
        {
            drawLine(canvas, shape[i][j], shape[i][j+1], pose, resolution, origin_x, origin_y, color);
        }
        drawLine(canvas, shape[i][0], shape[i][shape[0].size()-1], pose, resolution, origin_x, origin_y, color);
    }
```

Reply:
Any reason why `i++` is used instead of `++i`. That is the most common way to increment in for loops. Not just our code but all cpp.